packages/demand-deeplink/src/constants.ts

```diff
@@ -52,3 +54,9 @@ export const journeyLegMetaPrefixes: Record<string, string> = {
   meta: journeyLegMetaPrefix,
   passengerInfo: journeyLegMetaPrefix,
 }
+
+export enum BookingTypes {
+  ASAP = 'ASAP',
+  PREBOOK = 'PRE-BOOK',
+  CUSTOM = 'CUSTOM',
```

Comment:
Cpuld you please explain what is type custom?

Reply:
Yes, sure. In case if some unexpected type was provided in url string, e.g. `booking-type=wrong-type`, CUSTOM type will be added to deeplink (`bookingType: 'CUSTOM'`) and it will be considered as invalid during validation